pipeline/extract.py

```diff
@@ -191,8 +210,10 @@ def get_sales_data() -> list[dict]:
     logging.info("Sales data gathered")
     if data is not None:
         logging.info("Scraping begun")
+        print("Length of list:", len(data['feed_data']['events']))
         list_of_items = asyncio.run(
             extract_list_of_items(data['feed_data']['events']))
+        print("Item length: ", len(list_of_items))
```

Comment:
Shouldn't these prints be replaced by logging?

Reply:
Ah good point, those were sanity checks, but I'll delete both print statements with one log statement